config/main.py

```diff
@@ -5226,7 +5228,7 @@ def del_vrf(ctx, vrf_name):
     else:
         del_interface_bind_to_vrf(config_db, vrf_name)
         config_db.set_entry('VRF', vrf_name, None)
-
+        click.echo("VRF {} deleted and all associated IPv4 addresses removed.".format(vrf_name))
```

Comment:
We are deleting all IPs both IPv4 and IPv6 addresses. It would be more appropriate to change the message to "VRF {} deleted and all associated IP addresses removed"